tests/api_test.js

```diff
@@ -8,27 +8,31 @@ chai.use(chaiHTTP)
 describe("Test data collection api", function() {
 	var host = "http://" + "0.0.0.0" + ":" + 9090
 
-	it("POST to /api/v1.0/stats should return 200", function(done){
-		chai.request(host)
-			.post("/api/v1.0/stats")
-			.set("Content-Type", "application/json")
-			.send({myparam: "test"})
-			.end(function(err, res) {
-				res.should.have.status(200)
-				done()
-			})
-
+	describe("POST to /api/v1.0/stats", () => {
```

Comment:
Nice!!
Arrow function from ES6

Reply:
Thanks! 
You should try es6 related workshops at http://nodeshool.io

@mik-dass  should too.

At your own pace!